.ci/targets/windows_dart_unit_tests.yaml

```diff
@@ -2,4 +2,4 @@ tasks:
   - name: prepare tool
     script: .ci/scripts/prepare_tool.sh
   - name: custom package tests
```

Comment:
I was really confused by the failure link because this is wrong; please change to "dart unit tests".